setup.py

```diff
@@ -209,6 +213,7 @@
     + CASSANDRA_REQUIRED
     + AZURE_REQUIRED
     + ROCKSET_REQUIRED
+    + IKV_REQUIRED
```

Comment:
This is not necessary, since we are not adding tests to run on ci at this point

Reply:
thanks for flagging, removed